bees/jenkinsbee/jenkinsbee.go

```diff
@@ -153,6 +191,12 @@ func (mod *JenkinsBee) triggerBuild(jobname string) {
 	}
 }
 
+//func (mod *JenkinsBee ) pollLatestBuild() {
```

Comment:
If it's not used yet, remove it or add a comment why this code may be useful or will be required in the near future.